portfolio/src/main/webapp/photos-data.js

```diff
@@ -0,0 +1,45 @@
+export default [
```

Comment:
1. [Declare a type for Photo](https://google.github.io/styleguide/jsguide.html#features-declare-types-as-needed). Here's an example of doing so: https://jsdoc.app/tags-typedef.html
2. Declare this export to be an `!Array<!Photo>`.